db/db_impl.cc

```diff
@@ -161,6 +161,11 @@ DBImpl::DBImpl(const DBOptions& options, const std::string& dbname)
       write_buffer_manager_(immutable_db_options_.write_buffer_manager.get()),
       write_thread_(immutable_db_options_),
       write_controller_(mutable_db_options_.delayed_write_rate),
+      // Use delayed_write_rate as a base line to determine the initial
+      // low pri write rate limit. It may be adjusted later.
+      low_pri_write_rate_limiter_(NewGenericRateLimiter(std::min(
+          static_cast<int64_t>(mutable_db_options_.delayed_write_rate / 8),
+          static_cast<int64_t>(2 * 1024 * 1024)))),
```

Comment:
Wondering if it would be more readable if you pulled 2*1024*1024 into an appropriately named constant.